docs/source/llms/tracing/index.rst

```diff
@@ -706,6 +725,28 @@ and the :py:meth:`mlflow.client.MlflowClient.delete_trace_tag` method to remove
     client.delete_trace_tag(request_id="12345678", key="tag_key")
 
 
+Async Logging
+-------------
+
+By default, MLflow Traces are logged synchronously. This may introduce a performance overhead when logging Traces, especially when your MLflow Tracking Server is running on a remote server. If the performance overhead is a concern for you, you can enable **asynchronous logging** for tracing in MLflow 2.16.0 and later.
+
+To enable async logging for tracing, call :py:func:`mlflow.config.enable_async_logging` in your code. This will make the trace logging operation non-blocking and reduce the performance overhead.
+
+.. code-block:: python
+
+    import mlflow
+
+    mlflow.config.enable_async_logging()
+
+    # Traces will be logged asynchronously
+    with mlflow.start_span(name="foo") as span:
+        span.set_inputs({"a": 1})
+        span.set_outputs({"b": 2})
+
+
+Note that the async logging does not fully eliminate the performance overhead. Some backend calls still need to be made synchronously and there are other factors such as data serialization. However, async logging can significantly reduce the overall overhead of logging traces, empirically about ~80% for typical workloads.
```

Comment:
Is it possible to provide details on how we measured the reduction in overhead?

Reply:
Hmm I don't think users need that level of details here. This is just a dev environment so far less important than general benchmarking like database.

Reply:
Sounds good. I thought people might complain when they don't get 80% overhead reduction.